tests/e2e/replication/show.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+import typing
+import sys
+from itertools import zip_longest
+
+import pytest
+import mgclient
+from common import execute_and_fetch_all
+
+
+def test_show_replication_role(connection):
+    cursor = connection.cursor()
+    data = execute_and_fetch_all(cursor, "SHOW REPLICATION ROLE;")
+
+    assert cursor.description[0].name == "replication role"
+    assert data[0][0] == "main"
```

Comment:
Maybe make a connection to a replica and check their role also?

Reply:
Done please check it out again 😄